v2.1/admin-ui-user-authentication.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Enable User Authentication
+summary: Learn how to enable user authentication for secure clusters for the Admin UI.
+toc: false
+---
+
+By default, CockroachDB allows all users to access and view the Admin UI. For added security, you can choose to enable user authentication so that only authorized users can access and view the Admin UI.
```

Comment:
I'd suggest we call out that this is for secure cluster directly in the intro and remove the info callout:

> By default, CockroachDB allows all users to access the Admin UI. However, for secure clusters, you can enable user authentication so that only authorized users can access and view the Admin UI.